build_torcx_store

```diff
@@ -219,14 +219,14 @@ function torcx_package() {
 # swapping default package versions for different OS releases by reordering.
 DEFAULT_IMAGES=(
         =app-torcx/docker-1.12
-        =app-torcx/docker-17.10
+        =app-torcx/docker-17.09
 )
 
 # This list contains extra images which will be uploaded and included in the
 # generated manifest, but won't be included in the vendor store.
 EXTRA_IMAGES=(
 	=app-torcx/docker-17.03
-	=app-torcx/docker-17.09
+	=app-torcx/docker-17.10
```

Comment:
Should we drop CE Edge entirely for beta?  It doesn't seem likely to be used, especially from the Tectonic-specific bucket.

Reply:
Okay, dropped.